LipematWordpress/Sniffs/Sniff.php

```diff
@@ -0,0 +1,18 @@
+<?php
+/**
+ * WP phpcs Coding Standard.
+ *
+ * @package lipemat/wp-phpcs
+ * @link https://github.com/lipemat/wp-phpcs
+ * @license https://opensource.org/licenses/MIT MIT
+ */
```

Comment:
This doesn't really mean anything specific to this project. Is it required?

Reply:
Actually not, I was just following up on the pattern and comments of others sniffs structure, but we can remove it 👍